### ISSUE TITLE
Did not check() all arguments during internal call to 'lagConsole/setConfig'

### ISSUE THREAD

**Issue by tcastelli** [MasterAM/meteor-lag-console#3]
Getting this after app start

``` javascript
Error: Did not check() all arguments during internal call to 'lagConsole/setConfig'
W20151014-14:44:37.577(2)? (STDERR)     at [object Object]._.extend.throwUnlessAllArgumentsHaveBeenChecked (packages/check/match.js:357:1)
W20151014-14:44:37.578(2)? (STDERR)     at Object.Match._failIfArgumentsAreNotAllChecked (packages/check/match.js:112:1)
W20151014-14:44:37.578(2)? (STDERR)     at maybeAuditArgumentChecks (livedata_server.js:1689:18)
W20151014-14:44:37.578(2)? (STDERR)     at livedata_server.js:1605:18
W20151014-14:44:37.578(2)? (STDERR)     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
W20151014-14:44:37.578(2)? (STDERR)     at [object Object]._.extend.apply (livedata_server.js:1604:45)
W20151014-14:44:37.578(2)? (STDERR)     at [object Object]._.extend.call (livedata_server.js:1547:17)
W20151014-14:44:37.579(2)? (STDERR)     at configureConsole (packages/alon_lag-console/lib/server/lag_console_setup.js:35:1)
W20151014-14:44:37.579(2)? (STDERR)     at Package (packages/alon_lag-console/lib/server/lag_console_setup.js:39:1)
W20151014-14:44:37.579(2)? (STDERR)     at D:\xxx\.meteor\local\build\programs\server\packages\alon_lag-console.js:267:4
```

**Comment by MasterAM** [comment]
This is most likely due to the [auditargumentchecks](http://docs.meteor.com/#/full/auditargumentchecks) package. I will add a check to this method and release over the weekend.

Can you specify which audit packages you have installed? I would like to test against those.

Thanks for the report.

**Comment by MasterAM** [comment]
Released v1.0.1. Should now be fixed.
